apps/telehealth-widget/HOW-THE-DEMO-WORKS.md

```diff
@@ -0,0 +1,37 @@
+# How the Demo Works
+
+The solution uses [Amazon Chime SDK messaging](https://aws.amazon.com/blogs/business-productivity/integrate-your-identity-provider-with-amazon-chime-sdk-messaging/) to send messages for user status tracking and shares any metadata such as meeting information amongst users. The solution establishes a [MessagingSession](https://github.com/aws/amazon-chime-sdk-js#messaging-session) using Amazon Chime SDK for JavaScript to listen on to the [messages](https://docs.aws.amazon.com/chime-sdk/latest/dg/using-the-messaging-sdk.html#msg-types) the Amazon Chime SDK messaging sends.
+
+Note that, you can integrate your identity provider with Amazon Chime SDK messaging instead of using Amazon Cognito. You have to update the demo authentication components accordingly to use your identity provider. Please check [Integrate your Identity Provider with Amazon Chime SDK Messaging](https://aws.amazon.com/blogs/business-productivity/integrate-your-identity-provider-with-amazon-chime-sdk-messaging/) blog post for more information.
+
+
+## Sign in/Sign up
+
+The solution uses [Amazon Cognito](https://aws.amazon.com/cognito/) for authentication. When a user signs up, the solution creates an [Amazon Chime SDK Messaging](https://docs.aws.amazon.com/chime-sdk/latest/dg/using-the-messaging-sdk.html) `AppInstanceUser`. The solution sets up two Cognito groups in the Amazon Cognito user pool: a doctor group and a patient group. Based on the "Account Type," we assign the user to one of these groups. The authentication is built using the [Amplify UI library components](https://docs.amplify.aws/lib/auth/getting-started/q/platform/js/).
+
+
+## Appointment
+
+The doctor creates an appointment with a specific patient. The patient cannot create an appointment. The solution creates an Amazon Chime SDK messaging channel and adds two users as channel moderators. When a doctor creates a appointment, the appointment creation AWS Lambda function starts the [AWS step function](https://aws.amazon.com/step-functions/getting-started/) execution. The AWS step function waits for the appointment time and then it sends an SMS to the doctor and the patient in the appointment. The solution uses [Amazon Pinpoint](https://aws.amazon.com/pinpoint/) to send an SMS notification. We save the appointment date/time and doctor/patient information in the channel metadata.
```

Comment:
- `a appointment` => `an appointment`
- `appointment time and then it sends` => `appointment time, and then it sends` (Add a comma since a new sentence starts.)